vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectLinksComponent.java

```diff
@@ -50,11 +56,23 @@ public ProjectLinksComponent(@Autowired ProjectLinkingService projectLinkingServ
     Objects.requireNonNull(projectLinkingService);
     initOfferSearch(offerLookupService);
     initProjectLinksGrid();
-    getContent().addTitle(TITLE);
-    getContent().addContent(offerSearch, projectLinks);
+
+    initLayout();
+
     projectLinksComponentHandler = new ProjectLinksComponentHandler(projectLinkingService);
   }
 
+  private void initLayout() {
+    addClassName("attachments-area");
```

Comment:
NIT: I think the class name should go to the top of the constructor as it does not add to the structure of the component but rather to its style.